_data/tools.json

```diff
@@ -1934,5 +1934,14 @@
       "platforms": "Linux, MacOS, Windows",
       "note": "A vulnerability scanner for container images and filesystems.",
       "type": "SAST"
+   },
+    {
+      "title": "OWASP IDE VulScanner",
+      "url": "https://destinjidee.com/owasp-ide-vulscanner",
```

Comment:
If it's an OWASP project why doesn't it have an actual OWASP project page?

Reply:
Update the OWASP reference